app/javascript/src/components/form/LimitedCheckboxes.svelte

```diff
@@ -0,0 +1,54 @@
+<script>
+  export let name = "checkboxes"
+  export let categories = []
```

Comment:
Since this component is meant to be generic, this could probably be renamed to something like `items`

Reply:
Ah for sure!